plugins/org.locationtech.udig.project.tests/src/org/locationtech/udig/project/internal/render/impl/renderercreator/SingleFeatureRenderMetricsFactory2.java

```diff
@@ -0,0 +1,75 @@
+/* uDig - User Friendly Desktop Internet GIS client
+ * http://udig.refractions.net
+ * (C) 2004, Refractions Research Inc.
```

Comment:
Please use year of contribution

Reply:
OK I will adapt header comment in all new classes